frontend/src/app/containers/ExperimentPage.js

```diff
@@ -452,18 +454,18 @@ export class ExperimentDetail extends React.Component {
                   <div className="details-description">
                     <section className="graduation-report" dangerouslySetInnerHTML={createMarkup(graduation_report)}/>
                   </div>}
-              </div>
+              </LayoutWrapper>
             </div>
           </div>
-          {!hasAddon && <div>
-            <h2 className="card-list-header" data-l10n-id="otherExperiments">Try out these experiments as well</h2>
+          {!hasAddon && <Banner>
+            <h2 className="banner__subtitle centered" data-l10n-id="otherExperiments">Try out these experiments as well</h2>
             <div className="responsive-content-wrapper delayed-fade-in">
```

Comment:
@6a68 doing the layout wrapper thing...killing off most of the animations for now just to have a re-think about them